data/json/items/armor/torso_clothes.json

```
@@ -763,14 +763,14 @@
     "max_encumbrance": 5,
     "pocket_data": [ { "pocket_type": "CONTAINER", "max_contains_volume": "250 ml", "max_contains_weight": "1 kg", "moves": 80 } ],
     "warmth": 5,
-    "material_thickness": 1,
+    "material_thickness": 0.5,
     "flags": [ "VARSIZE", "FANCY" ]
   },
   {
     "id": "vest_leather",
     "type": "ARMOR",
     "name": { "str": "leather vest" },
-    "description": "A vest made from thick leather.  Offers excellent protection from cuts.",
+    "description": "A vest made from thick leather.",
```

```suggestion
    "description": "A vest made from leather.",
```
Possibly a `vest_thick_leather`, with greater thickness but more encumbrance & weight, could be added?

---

Actually I found a new source that says motorcycle jackets and similar leather items average 1.5mm, so I'm going to be revising this.